devops/poseidon-deployer/tasks/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { expect } from "chai";
+import "@nomiclabs/hardhat-ethers";
+
+const { buildPoseidonOpt } = require('circomlibjs');
+
+
+export const verifyPoseidonContracts = async (addresses: Array<any>) => {
+  const poseidonReference = await buildPoseidonOpt();
+  let poseidonContracts = new Map();
+  let input = [];
+
+  for (let TN = 2; TN <= 7; TN++) {
+    const address =addresses[`${TN-2}`];
+    poseidonContracts[`T${TN}`] = await ethers.getContractAt(`PoseidonT${TN}`, address);
+    input.push(`${TN}`);
+   
+    const expectedHash = poseidonReference.F.toString(poseidonReference(input));
```

Comment:
NIT: Add what the expect hashes/inputs actually are for each in comments

Reply:
Reflected:

https://github.com/privacy-scaling-explorations/maci/blob/3f753a1792931a5910c3614db2426bef2bb134c1/devops/poseidon-deployer/tasks/utils.ts#L40-L53